pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -1036,7 +1036,7 @@ public void testManagedCursorPersistStats() throws Exception {
         Multimap<String, Metric> metrics = parseMetrics(metricsStr);
 
         List<Metric> cm = (List<Metric>) metrics.get("pulsar_ml_cursor_persistLedgerSucceed");
-        assertEquals(cm.size(), 1);
+        assertEquals(cm.size(), 2);
```

Comment:
This became `2` because of the extra namespace metric. If you aggregated the `value` of each `Metric`, you will see that the metric is essentially double reported for the namespace.

Reply:
Yes, I will roll back as the reason mentioned above.